### PR TITLE
Remove references to CATS ASP module which is not present in OpenCATS

### LINKED ISSUE #247: Mass Import not working in 0.9.4 (testing)
Mass import displays the wrong page in 0.9.4 (testing).

### PR DESCRIPTION
**Pull request by mlespiau**
Remove references to modules/asp modules and dummy file. 
Fixes #247

### CODE REVIEW

**Comment by mlespiau** [comment]
@skrchnavy please review :)

File: modules/settings/SettingsUI.php [hunk 1]
```diff
@@ -681,10 +672,6 @@ public function handleRequest()
                 $this->getFirefoxModal();
                 break;
 
-            case 'downloads':
```
Comment (skrchnavy) [inline modules/settings/SettingsUI.php]: Is this really not used in non-asp part? I didn't check all code so just assure we are not removing more then necessary.

Comment (mlespiau) [inline modules/settings/SettingsUI.php]: I did not find any link to this page in the whole code base.

File: modules/settings/SettingsUI.php [hunk 2]
```diff
@@ -2456,7 +2443,7 @@ private function administration()
         $careerPortalUnlock = false;
         $careerPortalSettings = new CareerPortalSettings($this->_siteID);
         $cpData = $careerPortalSettings->getAll();
-        if (intval($cpData['enabled']) || (file_exists('modules/asp') && !$_SESSION['CATS']->isFree()) ||
+        if (intval($cpData['enabled']) || !$_SESSION['CATS']->isFree() ||
             LicenseUtility::isProfessional())
```
Comment (mlespiau) [inline modules/settings/SettingsUI.php]: file_exists('modules/asp') always evaluated to true - I just provided equivalent code at this point while removing the reference to modules/asp file.

Comment (skrchnavy) [inline modules/settings/SettingsUI.php]: I mean `LicenseUtility::isProfessional()` is valid check? Do we need it in the code of OPEN version?

**Comment by mlespiau** [comment]
We don't need it in the code but we should clean it up in another commit. I tried to that in the past but was not that simple.

**Pull request merged**